pull_request_template.md

```diff
@@ -1,19 +1,28 @@
+<!-- Comment in the sections you need for your PR! -->
+
+<!--
 ## 🎫 Ticket
 
 Link to the relevant ticket.
+-->
 
+<!--
 ## 🛠 Summary of changes
 
 Write a brief description of what you changed.
+-->
```

Comment:
IMO this section should almost always be filled out. How about something more like the following?

```suggestion
<!-- Update this summary if you'd like to provide a more detailed description of your changes. -->
## 🛠 Summary of changes

Check the pull request title for a description of what changed.
```

Reply:
I think that would still end up with PRs with the literal content "Check the pull request title for a description of what changed" which is what really bugs me

Reply:
Yeah, if the idea with the pull request is to find a solution to the problem of noise on the reviewer's end in anticipation that some people will inevitably submit the template without revisions, I don't think we'd want any uncommented text to remain in the template.

Reply:
Thanks! I think I'm going to leave the PR as-is, with all the text commented out, but we can always re-evaluate and update in the future as needed